shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -715,6 +730,11 @@ - (instancetype)initWithTarget:(id)target
   return self;
 }
 
+- (ForwardingGestureRecognizer*)recreateRecognizerWithTarget:(id)target {
```

Comment:
Would it be possible for `ForwardingGestureRecognizer` to instead implement `NSCopying`?

Reply:
Talked to @hellohuanlin, this isn't feasible because of the `move` which makes the old recognizer unusable.